scoring/src/main/scala/ai/h2o/sparkling/ml/internals/H2OMetric.java

```diff
@@ -43,7 +43,8 @@
   Withinss(false),
   Betweenss(true),
   TotWithinss(false),
-  Totss(false);
+  Totss(false),
+  MeanPerClassError(false);
```

Comment:
The boolean flag false in this case means: the higher value the metric has, the worse model we get.